thirdparty/README.md

```diff
@@ -17,9 +17,23 @@ TODO.
 
 ## fonts
 
+### Noto Sans UI Regular
+
+- Upstream: https://github.com/googlei18n/noto-fonts
+- Version: 1.06
+- License: OFL-1.1
+
+#### What’s the difference between the UI and non-UI versions?
```

Comment:
So do we use the UI or non-UI version? :)

Why is this comment relevant for us?

Reply:
we use UI version which has tighter vertical space.
I made this comment for somebody not to confuse which should be used between UI and Non-UI version in the future.